engine/dlib/src/dmsdk/external/remotery/Remotery.h

```diff
@@ -0,0 +1,1089 @@
+
```

Comment:
Due to the fact that the Remotery api uses defines, we need to have this header in dmsdk. But it is to be considered private.